### PR TITLE
Mark ExpandPersistentVolumes feature to beta

### PR DESCRIPTION
**Pull request by tengqm**
xref: kubernetes/kubernetes#64288

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -63,6 +63,8 @@ githubbranch = "v1.11.0"
 docsbranch = "release-1.11"
 url = "https://kubernetes.io"
 
+[[params.versions]]
+fullversion = "v1.10.3"
```
Comment (mistyhacks) [inline config.toml]: Did you change this on purpose? If not, can you revert? If so, can you let me know why?

Comment (tengqm) [reply config.toml]: Yes, this was changed on purpose. Problem found when I was rebasing my local release-1.11 branch to the upstream release-1.11 branch. The problem is not evident when rendered by github.

Before change:

```
 [[params.versions]]
 fullversion = "v1.11.0"
 version = "v1.11"
 githubbranch = "v1.11.0"
 docsbranch = "release-1.11"
 url = "https://kubernetes.io"

 version = "v1.10"
 githubbranch = "v1.10.3"
 docsbranch = "release-1.10"
 url = "https://kubernetes.io"
```

After change:

```
 [[params.versions]]
 fullversion = "v1.11.0"
 version = "v1.11"
 githubbranch = "v1.11.0"
 docsbranch = "release-1.11"
 url = "https://kubernetes.io"

 [[params.versions]]
 fullversion = "v1.10.3"
 version = "v1.10"
 githubbranch = "v1.10.3"
 docsbranch = "release-1.10"
 url = "https://kubernetes.io"
```

The difference is clearer, right?

Comment (mistyhacks) [reply config.toml]: @Bradamant3 @zacharysarah was this a missed step in the 1.10 release? Is this a new requirement with Hugo? I'm shaky on the `config.toml` syntax.

**Pull request merged**